src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -121,6 +123,8 @@ public class InstancesController : ControllerBase
             [FromQuery(Name = "visibleAfter")] string visibleAfter,
             [FromQuery] string dueBefore,
             [FromQuery] string excludeConfirmedBy,
+            [FromQuery(Name = "status.isSoftDeleted")] bool isSoftDeleted,
```

Comment:
Not exposing isHardDeleted because it didn't seem natural.. Any objections? They aren't filtered on the way out, so maybe the parameter should be included after all?

Reply:
I don't find it natural to expose isHardDeleted either. Hard deleted instances should probably be excluded.

Reply:
Excluding hard deleted instances